src/components/Popups/ReplyField/ItemList.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { SyntheticEvent } from 'react';
+import noop from 'lodash/noop';
+import DefaultItemRow from './DefaultItemRow';
+import { Collaborator } from '../../../@types';
+import './ItemList.scss';
+
+export type Props = {
+    activeItemIndex?: number;
+    filterString?: string;
+    itemRow?: JSX.Element;
+    items: Collaborator[];
+    onSelect: (index: number, event: React.SyntheticEvent) => void;
+    setActiveItemIndex?: (index: number) => void;
+};
+
+const ItemList = ({
+    activeItemIndex = 0,
+    itemRow = <DefaultItemRow />,
+    items,
+    onSelect,
+    setActiveItemIndex = noop,
+    ...rest
+}: Props): JSX.Element => {
+    return (
+        <ul role="listbox" {...rest}>
+            {items.map((item, index) => {
+                if (!item) {
+                    return null;
+                }
+
+                return React.cloneElement(itemRow, {
+                    ...item,
+                    key: item.id,
+                    className: 'ba-ItemList-row',
+                    isActive: index === activeItemIndex,
+                    onClick: (event: SyntheticEvent) => {
+                        onSelect(index, event);
+                    },
+                    /* preventDefault on mousedown so blur doesn't happen before click */
+                    onMouseDown: (event: SyntheticEvent) => {
+                        event.preventDefault();
+                    },
+                    onMouseEnter: () => {
+                        setActiveItemIndex(index);
```

Comment:
Would applying hover/focus css styles be preferred over actually changing the `activeItemIndex`?

Reply:
> Would applying hover/focus css styles be preferred over actually changing the `activeItemIndex`?

Yeah, they're in `DatalistItem`